### DISCUSSION TITLE
Support PowerShell 7.1 on macOS

### DISCUSSION THREAD

**Discussion by chrisallennc** [thycotic-ps/thycotic.secretserver#240] [Q&A : Module] [answered]
Do you have any plans to provide support to run the module with PowerShell 7.1 on macOS? We have a team member who is running a Mac and has issues when New-TssSession hits New-TssApiToken. It looks like the C# may just need to be compiled for macOS as long as it will work with .NET Core?

**Comment by wsmelton** [comment]
It currently supports use on macOS, Linux, and Windows with PowerSHell 7+.

**Comment by wsmelton** [comment, marked as answer]
Found issue, created #241. Will be fixed in the next release.

**Reply by chrisallennc** [reply]
We just tested and it's working, thanks!
